music_assistant/server/providers/plex/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 import asyncio
 import logging
+import requests
```

Comment:
verify if requests is part of the plex library, otherwise this import will fail.
we are not doing anything with requests but its async sibbling, aiohttp.

Reply:
I'm not sure how to check this.
Found this at least: https://github.com/pkkid/python-plexapi/blob/abcab4f78d22e86b380327c6014d3148e0248e6d/plexapi/server.py#L7

Reply:
Python plexapi has requests as its (sole) requirement, so the answer is yes.
https://github.com/pkkid/python-plexapi/blob/abcab4f78d22e86b380327c6014d3148e0248e6d/requirements.txt#L5